tw-NEDa-frontend/src/app/app.component.ts

```diff
@@ -1,27 +1,28 @@
-import { Component } from '@angular/core';
```

Comment:
Fix the formatting & code style. Once the formatter is the same this file should not appear as something that changed.